homeassistant/components/ecobee/switch.py

```diff
@@ -29,12 +30,17 @@ async def async_setup_entry(
     data: EcobeeData = hass.data[DOMAIN]
 
     async_add_entities(
-        (
-            EcobeeVentilator20MinSwitch(data, index)
+        [
```

Comment:
Note that list comps are fast than genexpr in python 3.12+ so there is no benefit to switching these to genexpr anymore